specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -11,7 +11,19 @@
         "configuration": {
           "ingress": {
             "external": true,
-            "targetPort": 3000
+            "targetPort": 3000,
+            "customDomains": [
```

Comment:
Adding a non-readonly property in an api version is considered a breaking change. Some existing clients ould get broken. 

Consider if you need to update the api-version, but this comment is not blocking. If you do not have too many clients and are willing to work with any existing customers to update thier broken clients if any then you can go ahead with this.

Reply:
@raosuhas we don't have any clients yet so we should be ok.